crates/ruff_linter/src/rules/flake8_pyi/rules/string_or_bytes_too_long.rs

```diff
@@ -67,3 +77,40 @@ pub(crate) fn string_or_bytes_too_long(checker: &mut Checker, string: StringLike
     )));
     checker.diagnostics.push(diagnostic);
 }
+
+fn warnings_dot_deprecated_message<'a>(
+    expr: Option<&'a ast::Expr>,
+    checker: &Checker,
```

Comment:
Nit: I'd recommend passing in `semantic: &SemanticModel` here, since it's a much lighter dependency than `&Checker` (and makes the function's dependencies much clearer).